operator/src/test/java/oracle/kubernetes/operator/rest/RestBackendImplTest.java

```diff
@@ -334,10 +411,86 @@ private void setupScanCache() {
     config = configSupport.createDomainConfig();
   }
 
+  @SuppressWarnings("unchecked")
+  public static <T> T getValue(Object object, String fieldName) throws IllegalAccessException {
+    return (T) getValue(object, getField(object.getClass(), fieldName));
+  }
+
+  private static Object getValue(Object object, Field field) throws IllegalAccessException {
+    boolean wasAccessible = field.isAccessible();
+    try {
+      field.setAccessible(true);
+      return field.get(object);
+    } finally {
+      field.setAccessible(wasAccessible);
+    }
+  }
+
+  private static Field getField(Class<?> aaClass, String fieldName) {
+    assert aaClass != null : "No such field '" + fieldName + "'";
+
+    try {
+      return aaClass.getDeclaredField(fieldName);
+    } catch (NoSuchFieldException e) {
+      return getField(aaClass.getSuperclass(), fieldName);
+    }
+  }
+
   private class TopologyRetrieverStub implements TopologyRetriever {
     @Override
     public WlsDomainConfig getWlsDomainConfig(String ns, String domainUid) {
       return config;
     }
   }
+
+  private class RestBackEndStub extends RestBackendImpl {
+
+    /**
+     * Construct a RestBackendImpl that is used to handle one WebLogic operator REST request.
+     *
+     * @param principal        is the name of the Kubernetes user to use when calling the Kubernetes
+     *                         REST api.
+     * @param accessToken      is the access token of the Kubernetes service account of the client
+     *                         calling the WebLogic operator REST api.
+     * @param domainNamespaces a list of Kubernetes namepaces that contain domains that the WebLogic
+     */
+    RestBackEndStub(String principal, String accessToken,
```

Comment:
Seems an odd place to break this method. Can it really not fit on one line?

Reply:
Fixed.